packages/core/useTransition/index.ts

```diff
@@ -90,14 +90,15 @@ export function useTransition(source: Ref<number>, options: TransitionOptions =
     ? transition
     : createEasingFunction(transition)
 
+  let currentDuration = 0
```

Comment:
```suggestion
  let currentDuration = unref(duration)
```

Do we need this here?

Reply:
@antfu This would be an unnecessary call to `unref`, and would have no effect on behavior. All of these variables are assigned their start values in the `watch` callback. I'm initializing them `0` just for the sake of casting to a number, we could do something like this if we wanted though...

```ts
let currentDuration: number
let diff: number
let endAt: number
let startAt: number
let startValue: number
```

But then they would all briefly have values of `undefined`. This wouldn't matter though since they all _will_ have values by the time the raf loop starts 🤷‍♂️ 

Whichever way you want to go here is fine with me, but I would recommend we call `unref` only once at the start of a transition.

Reply:
That makes sense to me, thanks!